gulp/tasks/lint.js

```diff
@@ -29,9 +19,17 @@ gulp.task( 'lint:gulp', function() {
 /**
  * Lints the source js files for errors
  */
-gulp.task( 'lint:src', function() {
+gulp.task( 'lint:js', function() {
```

Comment:
Good call. I failed at getting Less linting working, but it'd be nice to have that in the future (or PostCSS if we ever get there).